cds_ils/importer/serializers/schema.py

```diff
@@ -22,6 +22,7 @@ class ImportedEItemSchema(Schema):
     action = fields.String(dump_only=True)
     priority_provider = fields.Bool()
     duplicates = fields.List(fields.String)
+    ambiguous = fields.List(fields.String)
```

Comment:
I am a bit unsure what would be an ambiguous e-item in the context, let's chat about the details